components/Modal/WithdrawModal.tsx

```diff
@@ -45,7 +45,7 @@ export const WithdrawModal = (): JSX.Element => {
       }
 
       closeModal();
-      await updateCurrentUserBalance(currentUser.actor);
+      await updateAtomicBalance(currentUser.actor);
```

Comment:
We should update both the atomic market balance and the current user's balance.

Reply:
withdraw modal is no longer being used. Should we just get rid of it?

Reply:
Good point - yes!